src/docs/common_tasks/multiple_jvm_versions.md

```diff
@@ -0,0 +1,50 @@
+
+## Problem
+
+You want to compile and/or run different test or binary targets using different JVMs.
+
+Or, you need to specify command-line options globally for the JVM on a per-JVM version basis. For
+example, using `--add-opens` flags, but only when running on JVMs that support it.
```

Comment:
I don't see `--add-opens` option added in this PR, did I miss anything? Possibly some example would be helpful.

Reply:
Yes. That's a good idea. `--add-opens` is a jvm option that was added in jdk 9, so it's incompatible with 8.